lib/puppet/provider/cisco_ospf_area/cisco.rb

```diff
@@ -36,16 +43,22 @@
     :default_cost,
     :filter_list_in,
     :filter_list_out,
+    :nssa_route_map,
+    :nssa_translate_type7,
   ]
   OSPF_AREA_BOOL_PROPS = [
+    :nssa,
```

Comment:
I assume you reordered `OSPF_AREA_ALL_PROPS` below because you want `:nssa` to process before the other nssa properties? If so then I would add a comment; e.g.

```
  OSPF_AREA_BOOL_PROPS = [
    :nssa, # :nssa should process before other nssa properties
    :nssa_default_originate,
```